### DISCUSSION TITLE
Question about git sync image from branch to gitbook

### DISCUSSION THREAD

**Discussion by Zijian-Zhu** [GitbookIO/community#709] [3. Git Sync]
I have a few spaces that will use the same of images with different text in the page. So I separate them into multiple branches. But I am very confused when I try to add images.
 The first space I uploaded the images from the gitbook side and I saw the commit that let images stored in my first branch under .gitbook/assets. So I cherry pick this commit to my second branch which is gitsync with my second space. But I did not see any images under Select images after the git sync. Please tell me the right way to upload images for multiple spaces. Thank you.

**Comment by addisonschultz** [comment]
Hi @Zijian-Zhu - I'm not sure this will work. Can you try something for me? Can you try to upload an image in the second space, and let me know if it appears in the `.gitbook/assets` folder afterwards?

**Comment by Zijian-Zhu** [comment]
I uploaded an image in second space through gitbook UI. And that image appears in the `.gitbook/assets` folder with other images that cherry pick from first branch. I try a little bit yesterday about the image and I am guessing the way for handling image for gitbook UI and branch side are seperated. If I want to add image through gitbook ui, I have to upload image on gitbook. If I want to upload image using github, I have to work with markdown file. May you confirm my guess?

**Reply by addisonschultz** [reply]
@rodrcastro just want to double check to see if you've heard this one before!
